apps/readme.md

```diff
@@ -8,3 +8,92 @@ This directory contains different apps that are shipped with the SDK.
 | --------- | ----------- | -------------- |
 | server | C++ | Server application that allows clients to connect to camera attached to the target, control it and get frames from it |
 | uvc-app | C++ | Application that exposes the ToF system as a depth camera to various USB host systems and allows them to control it |
+
+# Enabling the apps on the NXP platform
+
+| :exclamation:  The server and uvc-app cannot be both enabled at the same time. |
+|-----------------------------------------|
+
+## server
+
+The server app is enabled by default. The app and the mass storage can be enabled by editing the usb-gadget.sh file located in
+/usr/share/systemd/usb-gadget.sh. 
+In order to enable this functionality you must uncomment the following lines:
+````
+create_rndis configs/c.1 rndis.0
+create_msd configs/c.1 mass_storage.0 /dev/mmcblk1p1
+````
+If you want to disable this service the lines must be commented.
+````
+#create_rndis configs/c.1 rndis.0
```

Comment:
this line will disable the server app from starting after system boots?

Reply:
No, it's for network over usb and maass storage. Updated.